### ISSUE TITLE
Asgard's Wrath load screens do not show in SteamVR

### ISSUE THREAD

**Issue by peteostro** [LibreVR/Revive#1347]
When running Asgard's wrath load screens show on your monitor but in the VR headset it shows default SteamVR environment until the level is loaded.

**Comment by forestgamer** [comment]
Try seeing how much storage you have left on your computer. If you have below 50-100GB left for your PC, loading screens may not show or you may be in the SteamVR menu as the game stutters during the loading screen. Try making a little extra space for this game (I know it's already a big download) and tell me if that limits the stuttering or fixes the issue entirely.

**Comment by Djneo134** [comment]
Same problem here. And the empty space has nothing to deal with it. I have 105 Gb of free space on my C drive and 170 Gb on the D (where the oculus folder is). This seems to be a Revive problem with Sanzaru games cause I have the same on Marvel Power United VR. Every loading screen become flashy and make the Steam VR environment appear. It's very annoying for the eyes (these flashs). This is worst during combat in Asgard's Wratgh. When I try to hit with the sword it makes a lag and stop the game during a short moment, and make the SteamVR environment appear. It's really none enjoyable to play.

We really need a Revive upgrade quickly...

**Comment by hookmanuk** [comment]
I have the same loading screen issue with 190Gb free on my windows drive and 400Gb free on the drive the game is installed on, disk space isn't the issue here.

I also have another issue where after some time (30 mins or so?), if I trigger any steam overlay, either the guardian boundary or fpsvr graphs, then the game turns into a slideshow until the overlay disappears. This is consistent every time I played, but only after a period of time has passed. 
My best guess is maybe a certain transition in game causes this to start happening?

Also something to note, I think everyone using revive has reprojection in the tavern hub, whereas apparently on native Oculus it performs a lot better.

**Comment by BrodoBaggins** [comment]
> Same problem here. And the empty space has nothing to deal with it. I have 105 Gb of free space on my C drive and 170 Gb on the D (where the oculus folder is). This seems to be a Revive problem with Sanzaru games cause I have the same on Marvel Power United VR. Every loading screen become flashy and make the Steam VR environment appear. It's very annoying for the eyes (these flashs). This is worst during combat in Asgard's Wrath. When I try to hit with the sword it makes a lag and stop the game during a short moment, and make the SteamVR environment appear. It's really none enjoyable to play.
> 
> We really need a Revive upgrade quickly...

+1

I have encountered a similar issue when setting off certain effects on anything but low settings specifically when throwing weapons, parrying attacks, or breaking barrels. I am using a Valve Index with an RTX 2080 + i5 9600 + 32gbs of 2400 mhz ram. Looking at Steam frametime charts it looks like the app just totally hangs for about a second then resumes normal play normally I'm seeing frametimes of about 6 ms then it will just drop about 30-80 frames in a row before resuming.

Playing on low seems to alleviate this issue, so I'm thinking it must be some specific visual effect being triggered that is not supported properly, but unfortunately it makes the game look a lot worse.

**Comment by zxsdklfhskyrvk** [comment]
Make sure you have updated your Nvidia drivers. https://www.vrfocus.com/2019/10/nvidias-latest-game-ready-driver-adds-support-for-asgards-wrath-vive-cosmos/

My load screens blink in and out.  If you look forward, you will see this.  If you turn any other direction you only see the SteamVR horizon background.

**Comment by Scope666** [comment]
I have the blink in and out issue also.  I usually open the Steam VR menu and click the Desktop button so I can at least read the hint.

**Comment by Armada651** [comment]
Please try this pre-release version: https://github.com/LibreVR/Revive/releases/tag/1.9.1

**Comment by Scope666** [comment]
Thanks!  This issue is fixed for me on Valve Index - SteamVR Beta 1.8.13 with 1.9.1 pre-release.

**Comment by aCID3D** [comment]
why closed ? still blinking :(

**Comment by zxsdklfhskyrvk** [comment]
This is fixed on the latest version.  Working for me.
